### DISCUSSION TITLE
CircularProgress different between CRA and Next

### DISCUSSION THREAD

**Discussion by diceroll123** [chakra-ui/chakra-ui#8616] [General] [answered]
### Description

I'm converting my create-react-app site to Next, and it's mostly fine, but I find that the CircularProgress' progress track loses its style in Next.

<img width="234" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/1243957/41b4a63f-5dcd-44b3-9205-d409e8be2593">

---

The difference in light mode is more jarring, and completely illegible if not for the blue progress indicator itself: 
<img width="227" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/1243957/dc2a54a9-c1d1-48d6-b048-a1074aea8695">

---

The track has the same chakra-generated css class on it for both, but only the one on CRA has this styling:
<img width="490" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/1243957/83d3ab9a-da6f-465a-b4e9-cb0941be349e">

The one in Next has no such class style, however it does have the class name attached to the circle element.

### Link to Reproduction

https://codesandbox.io/p/sandbox/sparkling-currying-2fn8s4

### Steps to reproduce

try the codesandbox in a Next setting (it's currently in CRA because the Next sandbox template seems broken)

It doesn't happen immediately, I have had to toggle the theme and refresh for it to happen. It usually looks fine on a clean build. Very strange.

### Chakra UI Version

2.8.2

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
`CircularProgress` is v2. Use `Progress.Circle` in v3. We are not shipping new v2 work.
